draft-ietf-quic-transport.md

```diff
@@ -3254,8 +3254,8 @@ The STREAM_ID_BLOCKED frame contains a single field.
 
 Stream ID:
 
-: A variable-length integer indicating the highest stream ID that the sender
-  was permitted to open.
+: A variable-length integer indicating the maximum stream ID of the streams that
```

Comment:
nit: I would remove "of the streams that" because I think it makes this sentence less clear.